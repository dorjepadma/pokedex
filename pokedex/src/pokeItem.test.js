import React from 'react';
import PokeItem from './pokeItem.js';
import renderer from 'react-test-renderer';

test('renders PokeItem.js correctly', () => {
  const tree = renderer
    .create(<PokeItem  pokemon={{}}/>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});