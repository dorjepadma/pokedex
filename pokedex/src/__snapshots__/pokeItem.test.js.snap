// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders PokeItem.js correctly 1`] = `
<li>
  <div
    className="info-container"
  >
    <h2 />
    <p
      className="type_1"
    />
    <p
      className="hp"
    />
  </div>
  <div
    className="url-image"
  >
    <img />
  </div>
</li>
`;
